src/cloud.c

```diff
@@ -502,6 +508,7 @@ int cloud_list_devices(void)
 					       MQ_EXCHANGE_CLOUD,
 					       MQ_CMD_DEVICE_LIST,
 					       headers, 1,
+					       0,
```

Comment:
Add a MQ_MSG_NO_EXPIRATION_TIME define, or something like that.

Reply:
I've added an comment